src/lib/Form/Factory/FormFactory.php

```diff
@@ -1202,6 +1202,8 @@ public function copyLocationSubtree(
     }
 
     /**
+     * @deprecated Deprecated in 1.1 and will be removed in 2.0.
```

Comment:
Hmm, why do we deprecate it? Bookmarks is a new feature shipped as of 2.2.

Reply:
If we start using REST to manage Location bookmark this will be dead code in admin ui. Therefore I mark controller, form type and form data as deprecated.

Reply:
Ok, so I simply remove dead code